packages/devtools_app/lib/src/screens/profiler/cpu_profile_model.dart

```diff
@@ -801,7 +801,7 @@ class CpuProfileMetaData extends ProfileMetaData {
   }
 }
 
-class CpuSampleEvent extends TraceEvent {
+class CpuSampleEvent extends ChromeTraceEvent {
```

Comment:
@bkonyi @derekxu16 general question - will the VM Service CPU profiler APIs also be moving away from the Chrome trace event format at some point?

Reply:
Maybe, but there's no plans to drop support currently on our timeline.